src/main/java/org/example/ListExamples.java

```diff
@@ -210,7 +210,19 @@ else if (num / ((values[k]) * 4) == 1) {
         }
         return bucket;
     }
-
+    public static List<Integer> bubbleSorting(List<Integer> list) {
```

Comment:
This method should be in a different package `algorithms.sorting.BubbleSort.java`